src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -88,8 +85,6 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
 
             CurrentConfiguration = configurationProvider.GetConfiguration();
 
-            SetBoundSolutionUIContext();
```

Comment:
I am pretty confident that removing this should be fine but maybe Im completely off

Reply:
I think it will cause a problem, unfortunately. The `BoundSolutionUIContext` is used to trigger the loading of the `ConnectedModePackage` and the `TaintSyncPackage`  (they both have `[ProvideAutoLoad(BoundSolutionUIContext.GuidString, PackageAutoLoadFlags.BackgroundLoad)]`).

A problem scenario would be the following:
1. user opens VS
 - none of our packages are loaded yet
2. user opens a bound solution
  - VS will start loading some of our packages in the background, but it will take some time and probably won't finish until after the solution has finished loading.

In particular, the `ActiveSolutionBoundTracker` could be loaded after the bound solution has finished opening. If `SetBoundSolutionUIContext()` isn't called in the constructor, the other packages won't load until the user opens/closes another solution.